src/models/two_nodes.js

```diff
@@ -0,0 +1,886 @@
+ * @property {number} qSensible  – Sensible heat loss from skin, [W/m2]
+ * @property {number} qSkin  – Total rate of heat loss from skin, [W/m2]. Equal to q_sensible + e_skin
+ * @property {number} qRes  – Total rate of heat loss through respiration, [W/m2]
+ * @property {number} tCore  – Core temperature, [°C]
+ * @property {number} tSkin  – Skin temperature, [°C]
+ * @property {number} mBl  – Skin blood flow, [kg/h/m2]
+ * @property {number} mRsw  – Rate at which regulatory sweat is generated, [kg/h/m2]
+ * @property {number} w  – Skin wettedness, adimensional. Ranges from 0 and 1.
+ * @property {number} w_max  – Skin wettedness (w) practical upper limit, adimensional. Ranges from 0 and 1.
+ * @property {number} set  – Standard Effective Temperature (SET)
+ * @property {number} et  – New Effective Temperature (ET)
+ * @property {number} pmvGagge  – PMV Gagge
+ * @property {number} pmvSet  – PMV SET
+ * @property {number} disc  – Thermal discomfort
+ * @property {number} tSens  – Predicted Thermal Sensation
+ * @public
+ */
+
+/**
+ * @typedef {Object} TwoNodesKwargs
+ * @property {boolean} [round=true]
+ * @property {boolean} [calculate_ce=false]
+ * @property {number} [max_sweating]
+ * @property {number} [w_max]
+ * @public
+ */
+
+/**
+ * @typedef {Object} TwoNodesKwargsRequired
+ * @property {boolean} round
+ * @property {boolean} calculate_ce
+ * @property {number} max_sweating
+ * @property {number} w_max
+ */
+
+/**
+ * 
+ * Two-node model of human temperature regulation:
+ * This model can be used to calculate a variety of indices, including:
+ * Gagge’s version of Fanger’s Predicted Mean Vote (PMV). This function uses
+ * the Fanger’s PMV equations but it replaces the heat loss and gain terms
+ * with those calculated by the two node model PMV SET and the predicted
+ * thermal sensation based on SET. This function is similar in all aspects
+ * to the pythermalcomfort.models.pmv_gagge(). However, it uses the
+ * pythermalcomfort.models.set() equation to calculate the dry heat loss by
+ * convection {@link #ref_25|[25]}.
+ * Thermal discomfort (DISC) as the relative thermoregulatory strain necessary
+ * to restore a state of comfort and thermal equilibrium by sweating. DISC is
+ * described numerically as: comfortable and pleasant (0), slightly
+ * uncomfortable but acceptable (1), uncomfortable and unpleasant (2),
+ * very uncomfortable (3), limited tolerance (4), and intolerable (S).
+ * The range of each category is ± 0.5 numerically. In the cold, the classical
+ * negative category descriptions used for Fanger’s PMV apply.
+ * Heat gains and losses via convection, radiation and conduction.
+ * The Standard Effective Temperature (SET)
+ * The New Effective Temperature (ET)
+ * The Predicted Thermal Sensation (TSENS)
+ * The Predicted Percent Dissatisfied Due to Draft (PD)
+ * Predicted Percent Satisfied With the Level of Air Movement” (PS)
+ * 
+ * @public
+ * @memberof models
```

Comment:
add `@docname` and the public name of the function for the docs (Gagge et al. two-node model)